content/backend/src/DomainName/LncdApp.DomainName.Services/DataAccess/Entities/AuthRole.cs

```diff
@@ -1,7 +1,6 @@
 using Microsoft.AspNetCore.Identity;
 
-namespace LncdApp.DomainName.Services.DataAccess.Entities
-{
-    public class AuthRole : IdentityRole<Guid>
-    { }
-}
+namespace LncdApp.DomainName.Services.DataAccess.Entities;
+
+public class AuthRole : IdentityRole<Guid>
```

Comment:
I'm starting to have doubts whether this class was useful at any point in any project.

Probably doesn't matter if we are about to get rid of ASP.NET Core Identity soon.

Reply:
It was required by the integration at one point (you had to specify the role type, but EF failed to handle `IdentityRole<Guid>` used directly).

Reply:
TIL